app/policies/project_policy.rb

```diff
@@ -21,13 +21,26 @@ def index?
     true
   end
 
+  def destroy?
+    admin? || record.maintainers.include?(user)
```

Comment:
Én az `index?` és a `new?` kivételével minden akciót megváltoztatnék arra, hogy csak akkor engedjük a felhasználónak, ha a `deactivated_at: nil`.

Ez azért lenne hasznos, mert a fenti formában meg tudna történni a következő:
1. User#1 bookmark-ol egy Project-et.
2. User#2 deaktiválja a Project-et
3. User#1 bookmark-kal megnyitja a Project-et
4. Megjelenik neki változatlanul a deaktivált Project, amit még update-elni is tud.